src/extension/control-flow/src/main/java/org/geoserver/flow/config/DefaultControlFlowConfigurator.java

```diff
@@ -107,19 +112,20 @@ public DefaultControlFlowConfigurator() {
         timeout = -1;
 
         Properties p = configFile.getProperties();
-        List<FlowController> newControllers = new ArrayList<FlowController>();
+        List<FlowController> newControllers = new ArrayList<>();
+        PriorityProvider priorityProvider = getPriorityProvider(p);
+        
         for (Object okey : p.keySet()) {
             String key = ((String) okey).trim();
             String value = (String) p.get(okey);
-            LOGGER.info("Loading control-flow configuration: " + key + "=" + value);
```

Comment:
So this has moved down to give more evaluable feedback right ?

Reply:
Yes, to avoid it logging loading of a directive that is not a flow controller